src/ontology/mondo-ingest.Makefile

```diff
@@ -13,6 +13,28 @@
 MAPPINGSDIR=../mappings
 SKIP_HUGE=false
 
+####################################
+### Package management #############
+####################################
+.PHONY: pip
+pip:
+	python3 -m pip install --upgrade pip setuptools
+
+.PHONY: oaklib
+oaklib: pip
+	 python3 -m pip install --upgrade $@
```

Comment:
Also changed these goals to use `$@` (\*shrug\*)

Reply:
I dont think this is needed/useful for `.PHONY` goals.. This makes more sense when $@ corresponds to a filename.